main/Makefile

```diff
@@ -0,0 +1,17 @@
+# Usage:
+# make          # compile all
+# make clean    # remote ALL binaries and objects
+
+# This is an example comment.
+
+# Defined compile to be used.  
+CC=gcc
+
+# Name of final binary.
+TARGET=supragma
+ 
+all:    main.o
+	$(CC) main.c -o $(TARGET)
+ 
+clean:
+	rm *.o $(TARGET)
```

Comment:
Maybe use $(RM) instead of rm

Reply:
https://www.cs.swarthmore.edu/~newhall/unixhelp/howto_makefiles.html

Reply:
Follows the same paradigm as the all command.

Reply:
Resolved